example/workflow/extension/package.json

```diff
@@ -60,21 +60,24 @@
         "enablement": "activeCustomEditorId == 'workflow.glspDiagram'"
       },
       {
-        "command": "workflow.goToNextNode",
-        "title": "Go to next Node",
+        "command": "workflow.goToPreviousNode",
```

Comment:
For some reason the navigation does not seem to work properly for me:
- The 'go to previous' is never even shown.
- The 'go to right' gets stuck quickly but that might be the expected behavior?
- The 'Show documentation...' does not work even if I have the 'Push' node selected.

Maybe something is off on my side, do you have any idea what that could be?